mandatory/2-function-creation.js

```diff
@@ -6,7 +6,10 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  let validNum = typeof num === "number" && num % 2 === 0 && num <= 100;
+  return validNum;
+}
```

Comment:
you can simplify this further by just returning line 10 and not creating a new variable